docs/hyproof/data-flow-diagrams.md

```diff
@@ -0,0 +1,33 @@
+# HyProof Data Flow Diagrams
+
+Scenario 01:
```

Comment:
I think this is wrong. The member must be added to the ledger before we can set an alias